Playbooks/playbook-Process_Email_-_Generic.yml

```diff
@@ -319,12 +328,117 @@ tasks:
           - operator: uniq
           - operator: Stringify
       emailkeywords: {}
-      emailmessageid: {}
+      emailmessageid:
+        complex:
+          root: Email
+          filters:
+          - - operator: match
+              left:
+                value:
+                  simple: Email.Headers
+                iscontext: true
+              right:
+                value:
+                  simple: '''Message-ID'', ''<.+>'''
+          accessor: Headers
+          transformers:
+          - operator: replaceMatch
+            args:
+              regex:
+                value:
+                  simple: '>''\)(.+)?'
+              replaceWith: {}
+          - operator: replaceMatch
+            args:
+              regex:
+                value:
+                  simple: .+\('Message-ID', '<
+              replaceWith: {}
       emailreceived: {}
-      emailreplyto: {}
-      emailreturnpath: {}
-      emailsenderip: {}
-      emailsize: {}
+      emailreplyto:
```

Comment:
On all those field, use Demisto filters and transformers to extract the fields. see EWSv2 mapping (you can just copy the logic)